docker/package/package_generator.py

```diff
@@ -66,6 +79,75 @@ def gen_control_file(pkg: Package, out):
     with open(out, 'w') as f:
         f.write(file_contents)
 
+def gen_spec_file(pkg: Package, out):
+    full_package_name = package.name if package.target_proto is None else f"{package.name}-{package.target_proto}"
+    build_requires = " ".join(build_deps + run_deps)
+    requires = " ".join(run_deps)
+    if pkg.systemd_unit is not None:
+        systemd_deps = "systemd systemd-rpm-macros"
+        systemd_install =f'''
+mkdir -p %{{buildroot}}/%{{_unitdir}}
+install -m 644 %{{name}}.service %{{buildroot}}/%{{_unitdir}}
+mkdir -p %{{buildroot}}/%{{_sysconfdir}}/default
+install -m 644 %{{name}}.default %{{buildroot}}/%{{_sysconfdir}}/default/%{{name}}
+install -m 0755 {package.systemd_unit.startup_script} %{{buildroot}}/%{{_bindir}}
+'''
+        systemd_files =f'''
+%{{_bindir}}/{package.systemd_unit.startup_script}
+%{{_unitdir}}/%{{name}}.service
+%{{_sysconfdir}}/default/%{{name}}
+'''
+        systemd_macros=f'''
+%post
+%systemd_post %{{name}}.service
+useradd tezos || true
```

Comment:
It creates a group with the same name as well, right? And does it create a home directory?

Reply:
> It creates a group with the same name as well, right?
Yes
> And does it create a home directory?
Yes, but unfortunately not in `/var/lib`, will update this place to use `/var/lib` instead of `/home` :(

Reply:
Updated